### ISSUE TITLE
Map selection behavior should match list selection behavior

### ISSUE THREAD

**Issue by dfoll** [Esri/coordinate-conversion-addin-dotnet#534]

**Comment by saip** [comment]
@dfoll Do we need deselection of points from map(click highlighted point from map) too ?

**Comment by dfoll** [comment]
@saip yes. when a point is highlighted on the map, clicking that point should remove the map selection and the list selection for the same point
